core/local/steps/initial_diff.js

```diff
@@ -122,6 +131,9 @@ async function initialDiff (buffer /*: Buffer */, out /*: Buffer */, pouch /*: P
               path: was.path
             })
           }
+        } else if (was && event.kind === 'file' && sameUpdateTime(was.updated_at, event.stats.mtime)) {
```

Comment:
`sameUpdateTime()` feels a bit weird, names says *update*, but takes a string and a date...
What about extracting the whole condition?

Reply:
It really says _update time_ ;-).
But I'll find something better that encompasses the whole condition.